akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -22,38 +22,10 @@ to see what this looks like in practice.
 
 ## Introduction
 
-Cluster sharding is useful when you need to distribute actors across several nodes in the cluster and want to
-be able to interact with them using their logical identifier, but without having to care about
-their physical location in the cluster, which might also change over time.
+For an introduction to Sharding concepts see @ref:[Cluster Sharding](typed/cluster-sharding.md). This documentation shows how to use the typed
```

Comment:
"This documentation" could be misleading. Might be interpreted as this page. I think that sentence can be removed.